manifests/server/config.pp

```diff
@@ -49,40 +49,40 @@
         user        => $user,
         auth_method => 'ident',
         auth_option => $local_auth_option,
-        order       => '001',
+        order       => 001,
```

Comment:
This is probably a bad pattern to get into. The leading 0 makes it octal, which is fine by this, but not for other numbers.

```
$ puppet apply -e 'notice(002)'
Notice: Scope(Class[main]): 2
Notice: Compiled catalog for ganymede.local in environment production in 0.15 seconds
Notice: Applied catalog in 0.03 seconds
$ puppet apply -e 'notice(009)'
Error: Could not parse for environment production: Not a valid octal number 009  at line 1:8 on node ganymede.local
```